sa/database.go

```diff
@@ -138,4 +138,6 @@ func initTables(dbMap *gorp.DbMap) {
 	dbMap.AddTableWithName(orderFQDNSet{}, "orderFqdnSets").SetKeys(true, "ID")
 	dbMap.AddTableWithName(authz2Model{}, "authz2").SetKeys(true, "ID")
 	dbMap.AddTableWithName(orderToAuthz2Model{}, "orderToAuthz2").SetKeys(false, "OrderID", "AuthzID")
+	dbMap.AddTableWithName(recordedSerialModel{}, "serials")
```

Comment:
Do these need a `.SetKeys()` invocation chained on the end? I can't remember the Goose-isms that matter here.

Reply:
SetKeys is required if you want to Get or Update by primary key: https://godoc.org/github.com/coopernurse/gorp#TableMap.SetKeys. It also lets you mark a primary key as autoincrement, which means you get back the assigned id when inserting. Since we don't Get or Update, we don't need the call (yet).

That said, having written that out I realize it's a big pothole to potentially step in later so I'll just add the call.